tests/unit/test_repo.py

```diff
@@ -88,6 +89,7 @@ def test_can_disable_repositories(self):
             open(other.filename).readlines(),
         )
 
+    @pytest.mark.skip("RepositoryMapping.add now calls apt-add-repository")
```

Comment:
Probably these tests should be removed completely, or have their case moved into integration tests, rather than skipping

Reply:
Yeah, let's remove them.